tests_recording/testbase.py

```diff
@@ -14,8 +15,8 @@
 from packit.local_project import LocalProject
 
 DATA_DIR = "test_data"
-PERSISTENT_DATA_PREFIX = os.path.join(
-    os.path.dirname(os.path.realpath(__file__)), DATA_DIR
+PERSISTENT_DATA_PREFIX = Path(
+    os.path.join(os.path.dirname(os.path.realpath(__file__)), DATA_DIR)
```

Comment:
@jscotka what about the possibility to use prefix as `Path` instance in requre as well?